api/health_check.proto

```diff
@@ -88,6 +88,14 @@ message HealthCheck {
   message RedisHealthCheck {
   }
 
+  // [#not-implemented-hide:] grpc.health.v1.Health-based healthcheck.
+  // See `GRPC doc <https://github.com/grpc/grpc/blob/master/doc/health-checking.md>`
+  // for details.
+  message GrpcHealthCheck {
+    // Name of the service to perform check on.
+    string service = 1;
```

Comment:
Envoy uses the term `cluster` for gRPC. Shall we be consistent with it?

Reply:
My idea was that this is what will be sent in `service` field in `HealthCheckRequest`:
https://github.com/grpc/grpc/blob/master/src/proto/grpc/health/v1/health.proto#L21. Would calling it `cluster` cause more confusion about how this maps on gRPC health protocol?